src/utils.rs

```diff
@@ -17,6 +17,7 @@ use crate::application::ArgTypes;
 lazy_static! {
 	static ref MAX_VOLUME: Mutex<u8> = Mutex::new(100_u8);
 	static ref DEVICE_NAME: Mutex<String> = Mutex::new("default".to_string());
+	static ref TOP_MARGIN: Mutex<f32> = Mutex::new(0.75_f32);
```

Comment:
With the `allocated_height()` change, the default height is a bit high compared to before, could you change the default to 0.85? That should be pretty similar :)